pom.xml

```diff
@@ -255,6 +255,7 @@
                     <version>${version.plugin.javadoc}</version>
                     <configuration>
                         <source>${version.java}</source>
+                        <sourcepath>src/main/java;target/generated-sources</sourcepath>
```

Comment:
With this change the aggregated javadocs do not build at all.

Reply:
Hmm, builds on my machine!

Reply:
When I run `mvn site` I do not see javadocs being generated:
```
$ git checkout 7921-javadocs
Switched to branch '7921-javadocs'
$ rm -rf target
$ mvn site
$ du -sh target/site/apidocs
du: target/site/apidocs: No such file or directory
```

If I remove the change in this PR then javadocs are generated.